### PR TITLE
Setup logging

### PR DESCRIPTION
**Pull request by sttts**
This PR sets up controller-runtime and klog logging. By default, klog verbosity is 3, but we filter by request throttling messages. With one or more of `-d` or `--debug`, or `--debug=<level>` verbosity can be increased. `-d -d -d` or `--debug=3` means klog verbosity `3+3`, to 6. With that (3), you see all the (client-go) http requests the binary is doing.

### CODE REVIEW

File: cmd/up/repository/repository.go [hunk 1]
```diff
@@ -46,9 +46,13 @@ func (c *Cmd) AfterApply(kongCtx *kong.Context) error {
 func PredictRepos() complete.Predictor {
 	return complete.PredictFunc(func(a complete.Args) (prediction []string) {
 		upCtx, err := upbound.NewFromFlags(upbound.Flags{})
+		upCtx.SetupLogging()
+
 		if err != nil {
 			return nil
 		}
+		upCtx.SetupLogging()
+
```
Comment (ezgidemirel) [inline cmd/up/repository/repository.go:55]: why do we call setup twice in this function?

**Review by ezgidemirel — APPROVED** [review]
added one comment, other than that LGTM

File: cmd/up/repository/repository.go (hunk 1 above)
Comment (sttts) [reply cmd/up/repository/repository.go:55]: good find. We shouldn't.

Comment (sttts) [reply cmd/up/repository/repository.go]: fixed

**Pull request merged**